docs/versioned_docs/version-3.0.x/faqs/cStor.md

```diff
@@ -0,0 +1,328 @@
+   
+   **Init:** cStor volume replica status init represents the volume is not yet created.
+   
+   **Recreate:** cStor volume replica status recreate represents an intermediate state before importing the volume(this can happen only when pool pod got restarted) in case of a non-ephemeral disk. If the disk is ephemeral then this status represents volume is going to recreate.
+   
+   **NewReplicaDegraded:** cStor volume replica is newly created and it make successful connection with the target pod. 
+   
+   **ReconstructingNewReplica:** cStor volume replica is newly created and it started reconstructing entire data from another healthy replica.
+
+
+### How to handle replicas with slow disks or slow connectivity in case of cStor volumes? {#slow-replicas-in-cstor-volumes}
+
+CStor target pod disconnects a replica if IO response is not received from a replica within 60 seconds. This can happen due to slow disks in cStor pools or slow connectivity between target pod and cStor pool pods. In order to allow tuning of IO wait time from its default value of 60 seconds, there is an environment variable IO_MAX_WAIT_TIME in `cstor-istgt` container of target pod.
+Add below kind of configuration in target pod deployment under `env` section of `cstor-istgt` container:
+
+```
+    env:
+    - name: IO_MAX_WAIT_TIME
+        value: 120
+```
+
+Please note that target pod gets restarts which can impact ongoing IOs.
+
+### How OpenEBS detects disks for creating cStor Pool?
+
+Any block disks available on the node (that can be listed with say `lsblk` ) will be discovered by OpenEBS. 
+Node Disk Manager(NDM) forms the BlockDevice CRs in the following way
+
+* Scan the list of disks.
+* Filter out the OS disks
+* Filter out any other disk patterns that are mentioned in `openebs-ndm-config` under `Configmap` in `openebs-operator.yaml`.
+
+NDM do some filtering on the disks to exclude, for example boot disk. By default, NDM excludes the following device path to create blockdevice CR. This configuration is added in `openebs-ndm-config` under `Configmap` in `openebs-operator.yaml` .
+
+```
+/dev/loop - loop devices.
+/dev/fd - file descriptors.
+/dev/sr - CD-ROM devices.
+/dev/ram - ramdisks.
+/dev/dm -lvm.
+/dev/md - multiple device ( software RAID devices).
+/dev/rbd - ceph block devices 
+/dev/zd - zfs volumes
+```
+
+It is also possible to customize by adding more disk types associated with your nodes. For example, used disks, unwanted disks and so on. This change must be done in the 'openebs-operator.yaml' file that you have downloaded before OpenEBS installation.
+
+**Example:**
+
+```
+  filterconfigs:
+    - key: path-filter
+      name: path filter
+      state: true
+      include: ""
+      exclude: "/dev/loop,/dev/fd0,/dev/sr0,/dev/ram,/dev/dm-,/dev/md,/dev/rbd,/dev/zd"
+```
+
+[Go to top](#top)
+### What is the difference between cStor Pool creation using manual method and auto method?
```

Comment:
```suggestion
[Go to top](#top)

### What is the difference between cStor Pool creation using manual method and auto method?
```

Reply:
The changes are made under- https://github.com/openebs/website/pull/293/commits/6e79209b1999a3142f783cea507f33d65d6ed31f